### PR TITLE
Update troubleshooting.mdx

### PR DESCRIPTION
**Pull request by gtelnet**
Updated directions for the snapshot load process, included manually blessed snapshot link and updated formatting.

### CODE REVIEW

**Comment by gtelnet** [comment]
@PharkMillups tagging you on another thanks!

File: docs/mine-hnt/validators/troubleshooting.mdx [hunk 1]
```diff
@@ -30,26 +30,28 @@ Syncing to the blockchain could be impacted for the following reasons:
...
+
+NOTE: If you build the Validator from source, the paths below may be different for you. For the following Docker example, `/var/data` is inside the docker container which is mapped to your `validator_data` directory on the host system and `validator_data` is in your $HOME directory.  
+
+* Use the API to determine the latest [automatically blessed snapshot](https://api.helium.io/v1/snapshots) from `https://api.helium.io/v1/snapshots` or the latest [manually blessed snapshot](https://snapshots.helium.wtf/mainnet/latest-snap.json) from `https://snapshots.helium.wtf/mainnet/latest-snap.json`
+* Choose the one with the highest block height and download the snapshot from Helium via:  
+`curl -O https://snapshots.helium.wtf/mainnet/snap-<height>`. If this snapshot is not available, try the next highest as the consensus group may have not agreed upon the snapshot.
 * Move that file into your `validator_data` directory.
 * Temporarily pause transaction sync via ```miner repair sync_pause```.
```
Comment (PaulVMo) [inline docs/mine-hnt/validators/troubleshooting.mdx:40]: You no longer need to sync_pause / sync_cancel and sync_resume. This can be removed.

**Review by jkowalcz — APPROVED** [review]
Looks like a better version of mine!

**Comment by PharkMillups** [comment]
Thanks! Merging.

**Review by PharkMillups — APPROVED** [review]

File: docs/mine-hnt/validators/troubleshooting.mdx [hunk 2]
```diff
@@ -30,26 +30,28 @@ Syncing to the blockchain could be impacted for the following reasons:
 * you just launched an AMI instance (if using AWS)
 
 ## Grab the latest blessed snapshot from Helium
-* On a running Validator run `miner snapshot list` or you can use the API to get the latest blessed snapshot going to this web address `https://api.helium.io/v1/snapshots`.
-* Find the highest "Height" block and download the snapshot from Helium via `curl -O https://snapshots.helium.wtf/mainnet/snap-<height>`. If this snapshot is not available, try the next highest as the consensus group may have not agreed upon the snapshot.
+
+NOTE: If you build the Validator from source, the paths below may be different for you. For the following Docker example, `/var/data` is inside the docker container which is mapped to your `validator_data` directory on the host system and `validator_data` is in your $HOME directory.  
+
+* Use the API to determine the latest [automatically blessed snapshot](https://api.helium.io/v1/snapshots) from `https://api.helium.io/v1/snapshots` or the latest [manually blessed snapshot](https://snapshots.helium.wtf/mainnet/latest-snap.json) from `https://snapshots.helium.wtf/mainnet/latest-snap.json`
```
Comment (PaulVMo) [inline docs/mine-hnt/validators/troubleshooting.mdx:36]: I suggest some additional clarification on the sources of the snapshots, maybe something like this:

Helium hosts snapshots for download for the convenience of syncing validators. You may download snapshot by following these steps
* First determine the height of the snapshot you would like to download:
  * The list of latest snapshot which were agreed upon by the consensus group is available on the [API](https://api.helium.io/v1/snapshots) from `https://api.helium.io/v1/snapshots`
  * Additionally, the latest snapshot used by hotspots to sync is also available from `https://snapshots.helium.wtf/mainnet/latest-snap.json`(https://snapshots.helium.wtf/mainnet/latest-snap.json). Note, these latest snapshot may include snapshots generated by Helium or the Helium ETL and are not agreed upon by the Consensus Group and may not have been tested to work on validators.
* `curl....`

File: docs/mine-hnt/validators/troubleshooting.mdx [hunk 3]
```diff
@@ -59,12 +61,13 @@ Besure to trust the Validator that you take your snapshot from, shenanigans coul
 * Move the `/var/data/snap-<height>` file into your root of the `validator_data` directory on the destination Validator.
 * Temporarily pause transaction sync via ```miner repair sync_pause```.
```
Comment (PaulVMo) [inline docs/mine-hnt/validators/troubleshooting.mdx:62]: No longer need the sync_pause / sync_cancle / sync_resume

**Pull request merged**